applications/OptimizationApplication/custom_utilities/response/linear_strain_energy_response_utils.cpp

```diff
@@ -218,22 +220,22 @@ void LinearStrainEnergyResponseUtils::CalculateStrainEnergyEntitySemiAnalyticSha
             pThreadLocalEntity->CalculateRightHandSide(rPerturbedRHS, r_process_info);
             r_initial_coordintes[0] -= Delta;
             r_coordinates[0] -= Delta;
-            AtomicAdd<double>(r_orig_node_sensitivity[0], 0.5 * inner_prod(rX, rPerturbedRHS - rRefRHS) / Delta);
+            AtomicAdd<double>(r_orig_node_sensitivity[0], inner_prod(rX, rPerturbedRHS - rRefRHS) / Delta);
```

Comment:
what is problem here?

Reply:
removed. sorry for not informing, this branch was a testing branch. I created the fresh branch in #12142